### DISCUSSION TITLE
What about the port to version 1.18.2?

### DISCUSSION THREAD

**Discussion by HuruMuru** [ejektaflex/Bountiful#228] [Mod Ideas]
Dear Bountiful mod developer, I would love to wait for your wonderful mod on Minecraft 1.18.2 It would be a great solution for many players who are waiting for your mod on past versions. Please consider this request. Respectfully, a fan of your mod.

**Comment by ejektaflex** [comment]
Hi there - It's not that I don't want to, it's more that I don't have time. However, this mod is open source and anyone is free to contribute their own free time towards doing the port - even you! PRs into this repository will be accepted as per the contribution guidelines included in the repo itself. Best of luck!

**Reply by HuruMuru** [reply]
Which programm can help with backporting?

**Reply by ejektaflex** [reply]
Here is information for Fabric:
https://fabricmc.net/develop/
https://fabricmc.net/wiki/tutorial:setup

**Reply by HuruMuru** [reply]
And what about forge?

**Reply by ejektaflex** [reply]
This is a common project using Architectury Loom. However, it barely relies on the Forge API at all and uses Yarn mappings from Fabric. Any changes you make to the common project will apply to both Fabric and Forge versions, and sometimes loader specific changes will need to be implemented. For more information, you'd have to visit their Discord.
